src/Exception/ProblemDetailsExceptionInterface.php

```diff
@@ -12,7 +12,7 @@
 /**
  * Defines an exception type for generating Problem Details.
  */
-interface ProblemDetailsException extends JsonSerializable
+interface ProblemDetailsExceptionInterface extends ExceptionInterface, JsonSerializable
```

Comment:
This interface should not extend the package marker interface; it's intended for _consumers_ to implement, and, as such, will fall within their own package inheritance.

The `ExceptionInterface` marker is for identifying which exceptions _are thrown by this package_.